packages/farcaster-js-neynar/src/v2/openapi/spec.yaml

```diff
@@ -145,6 +145,10 @@ components:
         - verifications
         - activeStatus
       properties:
+        object:
+          type: string
+          enum:
+            - user
```

Comment:
`User.object == 'user'`

Wild. Any clue why they added this?

Reply:
No idea. There are some quirks to the specs they generate.